scene/gui/range.cpp

```diff
@@ -184,6 +184,8 @@ double Range::get_as_ratio() const {
 
 	} else {
 
+		ERR_FAIL_COND_V_MSG(Math::is_equal_approx(get_max(), get_min()), 0.0, "Cannot get ratio when minimum and maximum value are equal.");
```

Comment:
As mentioned in my previous comment, the same problem happens in the previous branch if `get_min()` and `get_max()` are equal and non 0.